.circleci/config.yml

```
@@ -1,29 +1,51 @@
 version: 2
 jobs:
+  integration-php72-magento23:
+    docker:
+      - image: boltdev/m2-plugin-ci-php72:v2
```

including an auth here is how you would get private images

```suggestion
      - image: boltdev/m2-plugin-ci-php72:v2
        auth:
          username: $DOCKER_USERNAME
          password: $DOCKER_PASSWORD
```